pkg/kubectl/cmd/cp.go

```diff
@@ -124,10 +119,20 @@ func runCopy(f cmdutil.Factory, cmd *cobra.Command, out, cmderr io.Writer, args
 	if err != nil {
 		return err
 	}
-	if len(srcSpec.PodName) != 0 {
+
+	srcPodNameLen := len(srcSpec.PodName)
+	destPodNameLen := len(destSpec.PodName)
+	if srcPodNameLen != 0 && destPodNameLen != 0 {
+		if _, err := os.Stat(args[0]); err == nil {
```

Comment:
`args[0]`? Wrong?

Should be `srcSpec.File`, right?

Reply:
no, args[0] is right. if both look like remote file spec, i try to check if src exists in local filesystem and if it does, i treat it as a local file